src/core/lib/security/transport/security_connector.h

```diff
@@ -154,11 +144,10 @@ void grpc_channel_security_connector_check_call_host(
     const char *host, grpc_auth_context *auth_context,
     grpc_security_call_host_check_cb cb, void *user_data);
 
-/* Handshake. */
-void grpc_channel_security_connector_do_handshake(
+/* Registers handshakers with \a handshake_mgr. */
+void grpc_channel_security_connector_create_handshakers(
```

Comment:
how about `grpc_channel_security_connector_add_handshakers_to_manager`? (and change the function pointer below as well).